samples/matter/common/cmake/source_common.cmake

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# NORDIC SDK APP START
```

Comment:
You can change this comment to something else because actually, there is no NORDIC SDK APP START.